### ISSUE TITLE
Add non-pixelated fonts support

### ISSUE THREAD

**Issue by RobotoSkunk** [RobotoSkunk/PixelMan-Adventures-Legacy#4]
To improve accessibility, a button should be added in the configuration to disable pixelated fonts and replace them with the same original font.
